index.js

```diff
@@ -246,6 +246,11 @@ class SyntaxNode {
     cursor.tree = this.tree;
     return cursor;
   }
+
+  childNodeForFieldName(fieldName) {
```

Comment:
Could we rename it to `childByFieldName`? The same name is used in other bindings (at least in Rust and Python).